README.md

```diff
@@ -12,15 +11,30 @@ meteor add jazeee:spiderable-longer-timeout
 ```
 
 ### Things you must do to make it work
-#### Set `Meteor.isReadyForSpiderable` Flag
+#### Set `Meteor.isReadyForSpiderable` property
 Code will wait for a flag to be `true`, which gives finer control while content is preparing to be published.
 
 #### Optionally set `Spiderable.userAgentRegExps`
-`Spiderable.userAgentRegExps` is array of strings, of bot user agents that we want to serve statically, but do not obey the `_escaped_fragment_ protocol`.
+`Spiderable.userAgentRegExps` __{[*RegExp*]}__ - is array of Regular Expressions, of bot user agents that we want to serve statically, but do not obey the `_escaped_fragment_ protocol`.
 ```coffeescript
 Spiderable.userAgentRegExps.push /^vkShare/i
 ```
 
+#### Optionally set `Spiderable.cacheTTL`
+__Note:__ 
+ - Should be set before `Meteor.startup`
+ - Value should be {*Number*} in seconds
+ - To set new TTL you need to drop index on `createdAt_1`
+```coffeescript
+Spiderable.cacheTTL = 3600 # 1 hour in seconds
```

Comment:
I don't like acronyms or uncommonly used terms. Many people don't know what TTL is.
In addition, we should do this in minutes.  In reality, most people want 30 minutes or longer TTL.
Let's rename this to cacheLifetimeInMinutes